### PR TITLE
fix the the dialog box error and the user talk issue

### PR DESCRIPTION
**Pull request by sherlockbeard**
#2827 is fixed with this . the title with drop down is show only when the user is trying delete his own upload . and the dialog with text appear when user is trying to nominate someone else picture.

### CODE REVIEW

**Comment by sherlockbeard** [comment]
fixed the #2703. as they both were interdependent.
test the fix of user talk in production . because the beta mediawiki which is in debug mode doesn't support nomination. only the stable version of mediawiki supports it. 
@nicolas-raoul

**Comment by nicolas-raoul** [comment]
When you are finished testing, please provide a link to the talk page(s) that were edited by the app, thanks :-)

**Comment by sherlockbeard** [comment]
https://commons.wikimedia.org/wiki/User_talk:Sherlocktest  (uploader)
https://commons.wikimedia.org/wiki/User_talk:Sherlocktest2 (nominator )

File: app/src/main/java/fr/free/nrw/commons/delete/DeleteHelper.java [hunk 1]
```diff
@@ -110,7 +110,8 @@ private boolean delete(Media media, String reason) {
             mwApi.appendEdit(editToken, logPageString + "\n",
                     "Commons:Deletion_requests/" + date, summary);
             mwApi.appendEdit(editToken, userPageString + "\n",
-                    "User_Talk:" + sessionManager.getCurrentAccount().name, summary);
+                        "User_Talk:" + media.getCreator(), summary);
+
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/delete/DeleteHelper.java]: Good :-)

File: app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java [hunk 2]
```diff
@@ -366,66 +367,68 @@ public void onCopyWikicodeClicked(){
...
-        });
-        AlertDialog d = alert.create();
-        input.addTextChangedListener(new TextWatcher() {
-            private void handleText() {
-                final Button okButton = d.getButton(AlertDialog.BUTTON_POSITIVE);
-                if (input.getText().length() == 0) {
-                    okButton.setEnabled(false);
-                } else {
-                    okButton.setEnabled(true);
+        else {
```
Comment (neslihanturan) [inline app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java]: Same thing @nicolas-raoul pointed happened here too. Indentation of several lines are changed. Reviewers click "files changed" tab on your pull request page and see changed lines. When you made additional changes it is hard to understand for us what will be inserted into our code base if we merge this changes. Even if your additional changes are beneficial for code quality etc. please send them in separate pull request. We should be seeing only relevant lines changed under a single pull request.

**Comment by sherlockbeard** [comment]
@nicolas-raoul  @neslihanturan i have created a new pr with all the changes requested . #3014

**Pull request closed without merging**